SpatialConvolution.lua

```diff
@@ -284,6 +284,7 @@ function SpatialConvolution:clearDesc()
     self.oDescForBias = nil
     self.oSize = nil
     self.scaleT = nil
+    nn.utils.clear(self, '_input', '_gradOutput', 'input_slice', 'output_slice')
```

Comment:
I think it makes more sense to clear the intermediate tensors in clearState than in clearDesc. Otherwise clearDesc becomes a misnomer.